src/Compilers/CSharp/Portable/Symbols/Source/ExtensionGroupingInfo.cs

```diff
@@ -793,6 +793,44 @@ protected override IEnumerable<ICustomAttribute> GetAttributes(EmitContext conte
                 {
                     yield return extensionAttribute;
                 }
+
+                if (synthesizeDefaultMemberAttributeIfNeeded(ExtensionMarkerTypes) is { } defaultMemberAttribute)
+                {
+                    yield return defaultMemberAttribute;
+                }
+
+                static SynthesizedAttributeData? synthesizeDefaultMemberAttributeIfNeeded(ImmutableArray<ExtensionMarkerType> extensionMarkerTypes)
```

Comment:
This implementation is valid only under condition that all indexers use the same metadata name. I think that right now this is enforced only by the fact that we disallow usage of "IndexerName" attribute on extension indexers. But we haven't made the final design decision on that. If we start supporting the attribute, then we either must enforce (not assume) the same metadata name for all indexers in the group, or the indexer name must become another part of the grouping name.